save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -275,7 +240,47 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
             tr {
                 th {
                     colSpan = tableInstance.columns.size
-                    testExecutionFiltersRow()
+                    testExecutionFiltersRow {
+                        filters = state.filters
+                        onChangeFilters = { filterValue ->
+                            if (filterValue.status == null || filterValue.status?.name == "ANY") {
+                                setState {
+                                    filters.status = null
```

Comment:
Does this actually work? You are not updating state, you are mutating a field of state, and React doesn't trigger updates on it

Reply:
Okay, it works, see https://github.com/saveourtool/save-cloud/pull/925#discussion_r926782898